pyproject.toml

```diff
@@ -31,3 +27,34 @@ module = [
   "paramiko.*",
 ]
 ignore_missing_imports = true
+
+[tool.ruff]
+line-length = 79
+
+[tool.ruff.format]
+docstring-code-format = true
+
+[tool.ruff.lint]
+extend-select = [
+  "D",  # pydocstyle
+  "I",  # isort
+  "PL",  # pylint
+]
+
+# Commented rules were not available in ruff. Enable them as they are added to ruff
+# See: https://github.com/astral-sh/ruff/issues/970
+ignore = [
+  "N815",  # Invalid name
+  "SIM300",  # misplaced-comparison-constant
+  # "PLC0209",  # Formatting a regular string which could be a f-string (consider-using-f-string)
```

Comment:
Were the commented lines carried over from flake8? Can we remove them now?

Reply:
Those were not yet available pylint rules, see the header comment. This is not needed anymore as we reverted to keep pylint.